data/metadata.json

```diff
@@ -2,8 +2,7 @@
     "clubhouse/entry/hover": {
         "sound-file": "clubhouse/clubhouse_button_hover.webm",
         "loop": true,
-        "fade-in": 0,
-        "fade-out": 0
+        "bg": true
```

Comment:
Has this been discussed anywhere? This will have the effect of fading in and out the background music even if you momentarily hover over the button by accident.

Reply:
It was an error. I shouldn't had to modify that.